ui/deathknight/presets.ts

```diff
@@ -0,0 +1,140 @@
+import { Consumes } from '/wotlk/core/proto/common.js';
+import { EquipmentSpec } from '/wotlk/core/proto/common.js';
+import { Flask } from '/wotlk/core/proto/common.js';
+import { Food } from '/wotlk/core/proto/common.js';
+import { ItemSpec } from '/wotlk/core/proto/common.js';
+import { Potions } from '/wotlk/core/proto/common.js';
+import { Spec } from '/wotlk/core/proto/common.js';
+import { WeaponImbue } from '/wotlk/core/proto/common.js';
+import { Faction } from '/wotlk/core/proto/common.js';
+import { Player } from '/wotlk/core/player.js';
+
+import {
+	DeathKnightTalents as DeathKnightTalents,
+	DeathKnight,
+	DeathKnight_Rotation as DeathKnightRotation,
+	DeathKnight_Options as DeathKnightOptions,
+} from '/wotlk/core/proto/deathknight.js';
+
+import * as Enchants from '/wotlk/core/constants/enchants.js';
+import * as Gems from '/wotlk/core/proto_utils/gems.js';
+import * as Tooltips from '/wotlk/core/constants/tooltips.js';
+
+// Preset options for this spec.
+// Eventually we will import these values for the raid sim too, so its good to
+// keep them in a separate file.
+
+// Default talents. Uses the wowhead calculator format, make the talents on
+// https://wotlk.wowhead.com/talent-calc and copy the numbers in the url.
+export const FrostTalents = {
+	name: 'Frost Dps',
+	data: '23050005-32005350352203012300033101351',
+};
+
+export const DefaultRotation = DeathKnightRotation.create({
+	useScourgeStrike: false,
+});
+
+export const DefaultOptions = DeathKnightOptions.create({
+	dualWhield: true,
```

Comment:
I think this field is spelled wrong; also dual wielding is implicit depending on whether the user selected 2 weapons, so you don't need an option for it.